app/javascript/time_helpers.js

```diff
@@ -0,0 +1,6 @@
+export function formatMilliseconds(ms) {
+  const totalSeconds = Math.round(ms / 1000);
+  const minutes = Math.round(totalSeconds / 60);
```

Comment:
there is a rounding error here. my timeouts are 1 minute and 2 minutes:
```
warningTimeoutMs = 60000;
sessionTimeoutMs = 120000;
```
the countdown timer briefly starts at 1:00, then goes up to 1:59, then right at 1:30 remaining it goes back down to 0:29. Using `Math.floor()` fixes it.
```suggestion
  const minutes = Math.floor(totalSeconds / 60);
```